python/Ganga/Core/GangaRepository/GangaRepositoryXML.py

```diff
@@ -710,7 +713,11 @@ def _safe_flush_xml(self, this_id):
                 for idn in os.listdir(os.path.dirname(fn)):
                     split_cache = getattr(obj, self.sub_split)
                     if idn.isdigit() and int(idn) >= len(split_cache):
-                        rmrf(os.path.join(os.path.dirname(fn), idn))
+                        try:
+                            rmrf(os.path.join(os.path.dirname(fn), idn))
+                        except Exception as err:
```

Comment:
What are the expected exceptions then? I'll add the ones I can see but there are likely quite a few here